code/core/src/components/components/Button/Button.tsx

```diff
@@ -243,6 +243,8 @@ const StyledButton = styled('button', {
 
   '> svg': {
     animation:
-      animating && animation !== 'none' ? `${theme.animation[animation]} 1000ms ease-out` : '',
+      animating && animation !== 'none'
+        ? `${theme.animation[animation as 'glow']} 1000ms ease-out`
+        : '',
   },
```

Comment:
```suggestion
      animating && animation !== 'none'
        ? `${theme.animation[animation]} 1000ms ease-out`
        : '',
  },
```
And adjust line `137` by applying `none` as a default value for `animation`